be/Cf.WebApi/Utilities/ServicesConfiguration/ServiceCollectionExtensions.cs

```diff
@@ -81,6 +81,18 @@ public static void ConfigureServices(this IServiceCollection services, IConfigur
 
             c.OperationFilter<SecurityRequirementsOperationFilter>();
         });
+
+        var MyAllowSpecificOrigins = "_myAllowSpecificOrigins";

```

Comment:
Let's use camel case here

Also same here. `ConfigureServices` does have `Configuration` as a parameter so you can get the value from appsettings